Makefile

```diff
@@ -1,44 +1,9 @@
 PYTHON ?= python
 
-.PHONY: all style-check type-check clean clean-pyc clean-patchfiles clean-backupfiles \
-        clean-generated pylint reindent test covertest build
```

Comment:
why removing these one? we are using some targets on release process manually.

Reply:
This is a mistake. I'll fix this now.

Reply:
`make style-check` is used on our release-checklist. So please update `utils/release-checklist` also.

Reply:
@shimizukawa do you use these commands locally? I don't use them locally, so I don't object to remove them, but I worry about somebody uses it.

Reply:
@tk0miya I've updated the `utils/release-checklist` document now.
@shimizukawa  Please note, as mentioned above, that all but one of the targets I've removed has a (superior) replacement option in tox, and only the useless `reindent` tool is removed. Hopefully if you do you use these, migrating will be possible

Reply:
Note: we need to notify removing these targets to all developers. I don't know who uses them. The maintainers are not only shimizukawa.